blocks/library/image/style.scss

```diff
@@ -9,11 +9,19 @@
 	&[data-align="left"] {
 		float: left;
 		margin-right: $block-padding;
+
+		@include break-medium() {
+			margin-left: calc( 50% - #{ $visual-editor-max-width / 2 } );
+		}
```

Comment:
Should we move the visual styles out as part of this PR, or as part of for example #729? Tackling that one seems like a good successor to this one, as video should also have full width as an option.